docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _configuring-slack-bot:
+
+====================================
+Configure Slack bot for Spidermon
+====================================
+
+What are bots?
+==============
+
+A bot is a type of Slack App designed to interact with users via conversation.
+
+To work with `Slack Actions <https://spidermon.readthedocs.io/en/latest/actions.html#slack-action>`_, you will need a Slack bot which would send `notifications <https://spidermon.readthedocs.io/en/latest/getting-started.html#slack-notifications>`_ to your Slack workplace from Spidermon.
+
+Steps
+=====
+
+.. note:
+    You would need to be leader of Slack workplace for which you are trying to create a bot.
```

Comment:
```suggestion
    You must be a leader of the target Slack workplace.
```
- ‘a leader’ → ‘the leader’ if there can be only one leader.
- It would be great if ‘leader’ could link to Slack documentation about making someone a leader of a workplace, or at least general documentation about Slack leaders. Similarly, ‘Slack workplace’ should point to documentation that explains what a Slack workplace is, or how to create one.
- I haven’t seen the work ‘leader’ in the Slack documentation. We should use the precise wording they use in Slack. Admin? Owner?

Reply:
Old documentation for Slack bots refer admins to be leaders, that's where I picked it up from. 
Changed to admins, and added links.